package.json

```diff
@@ -143,13 +144,73 @@
                 "category": "Azure Resource Manager Tools",
                 "title": "Reload Cached Schemas",
                 "command": "azurerm-vscode-tools.reloadSchemas"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort template",
```

Comment:
"Sort Template..."

The "..." indicates that the user will be asked for input before performing the action.

Reply:
Fixed.